packages/amplication-data-service-generator-runner/Dockerfile

```diff
@@ -1,16 +1,6 @@
 FROM node:19.1-alpine AS base
 RUN apk add g++ make python3
 
-WORKDIR /workspace/packages/amplication-code-gen-types
-COPY ./dist/packages/amplication-code-gen-types/package.json .
-RUN npm i --omit-dev
-COPY ./dist/packages/amplication-code-gen-types .
-
-WORKDIR /workspace/libs/data-service-generator
-COPY ./dist/libs/data-service-generator/package.json .
-RUN npm i --omit-dev
-COPY ./dist/libs/data-service-generator .
-
```

Comment:
why did you remove it?

Reply:
This is not required any more because we installing the DSG as a npm package

Reply:
and what about the code-gen?

Reply:
Its come as a dependency of the DSG